templates/support_channels.html

```diff
@@ -6,295 +6,46 @@
-						ANNOUNCEMENTS_SUPPORT_DESCRIPTION -- {{ brand }}
-					{% endblocktranslate %}
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="https://status.cs.aalto.fi/d/r0N6S9jik/main?orgId=1">
-						{% translate "STATUS" %}
-					</a>
-				</p>
-				<p class="card-text">
-					{% translate "CHECK_STATUS_OF_SERVERS" %}
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="https://wiki.aalto.fi/display/EDIT/Roadmap+2020-2024">
-						{% translate "CURRENT_DEVELOPMENT" %}
-					</a>
-				</p>
-				<p class="card-text">
-					{% blocktranslate trimmed %}
-						CURRENT_DEVELOPMENT_SUPPORT_DESCRIPTION -- {{ brand }}
-					{% endblocktranslate %}
-				</p>
-			</div>
-		</div>
-
-		<div class="card frontpage">
-			<div class="card-body">
-				<h3>{% blocktranslate trimmed %}OTHER_SERVERS_THAT_USE -- {{ brand }}{% endblocktranslate %}</h3>
-				<p class="card-text">
-					<a target="_blank" rel="noopener"
-						href="https://wiki.aalto.fi/display/EDIT/Exam+studio+exam+with+Aplus+exercises">
-						{% translate "EXAM_SERVER_FOR_EXAM_STUDIO" %}
-					</a>
-				</p>
-				<p class="card-text">
-					{% blocktranslate trimmed %}
-						EXAM_SERVER_SUPPORT_DESCRIPTION -- {{ brand }}
-					{% endblocktranslate %}
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="https://minus.cs.aalto.fi/">
-						{% translate "TESTING_ENVIRONMENT" %}
-					</a>
-				</p>
-				<p class="card-text">
-					{% blocktranslate trimmed %}
-						TESTING_ENVIRONMENT_SUPPORT_DESCRIPTION
-					{% endblocktranslate %}
-				</p>
-			</div>
-		</div>
-
-		<div class="card frontpage">
+		<div class="widecard frontpage">
 			<div class="card-body">
-				<h3>{% translate "LEARNING" %}</h3>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="https://wiki.aalto.fi/display/EDIT/Accessible+Teaching">
-						{% translate "ACCESSIBLE_TEACHING" %}
-					</a>
-				</p>
-				<p class="card-text">
-					{% translate "ACCESSIBLE_TEACHING_SUPPORT_DESCRIPTION" %}
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener"
-						href="https://aalto.cloud.panopto.eu/Panopto/Pages/Sessions/List.aspx#folderID=%2253f1b55a-0cb3-4c71-8302-abbf00c1a6d2%22">
-						{% translate "WATCH_WEBINAR_RECORDINGS" %}
-					</a>
-				</p>
-				<p class="card-text">
-					{% translate "WATCH_WEBINAR_RECORDINGS_SUPPORT_DESCRIPTION" %}
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="https://www.youtube.com/watch?v=2DG4feWn_Ws">
-						{% translate "INTRO_TO_A+_ÄLYOPPI_WEBINAR_IN_FINNISH" %}
-					</a>
-				</p>
+				<h3>{% translate "STUDENTS" %}</h3>
 				<p class="card-text">
 					{% blocktranslate trimmed %}
-						ÄLYOPPI_WEBINARS_AVAILABLE_AT_PROJECT_WEBSITE
+					CONTACT_COURSE_STAFF_FOR_ASSISTANCE_ON_COURSE_MATTERS
 					{% endblocktranslate %}
 				</p>
 			</div>
 		</div>
 	</div>
 </section>
+
 <section class="text-center">
```

Comment:
Sections must be labeled in WebCAG: `aria-label` or `aria-labelled-by`.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label